readme.md

```diff
@@ -157,6 +157,21 @@ Use this command to compile the `problem.pdf` from the `problem_statement/proble
 
 This can also be used to create the contest pdf by running it from the contest directory.
 
+## Personal configuration file
+
+For some command-line flags, it is convenient if they are always set to the same value, which differs per user
+(e.g., `--username` or `--password` for commands that access a CCS like DOMjudge,
+or `--scoreboard_repo` for the `bt solvestats` command).
```

Comment:
```suggestion
or `--scoreboard-repo` for the `bt solvestats` command).
```

Reply:
I see you made the hyphens/underscores consistent, nice :smile: (57be5a8)